### ISSUE TITLE
Table is no longer usable for screen reader users with Jaws 2026

### ISSUE THREAD

**Issue by havvamehfooz** [FreedomScientific/standards-support#947]
Summary

Table is no longer usable for screen reader users with jaws 2026 which was working perfectly fine with Jaws 2025

Example:

Attached the snippix file

[JAWS regression missing announcements for grid.html](https://github.com/user-attachments/files/29037289/JAWS.regression.missing.announcements.for.grid.html)
[JAWS regression missing announcements for table.html](https://github.com/user-attachments/files/29037290/JAWS.regression.missing.announcements.for.table.html)

Issue:

We recently found a couple of regression issues in JAWS that heavily impact the accessibility of the tables. 

In the past there were some effort to get rid of custom announcements in the table and we currently rely on JAWS announcements for the row and column count as well as the column headers. This was working perfectly with JAWS 2025, no missing or double announcements. Unfortunately JAWS 2026 doesn't announce these information anymore, just the cell content. With that the table is not usable for JAWS users. We haven't done any related changes meanwhile and NVDA announces the information as expected. Please find attached a plain html example for testing and for the report.

**Comment by JeremiahRogersVispero** [comment]
@havvamehfooz, thanks for this post. As noted in comments to #946, the change in row and column announcement when navigating among focusable table cells was intentional.
